modules/ztp-querying-the-policy-compliance-status-for-each-cluster.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-querying-the-policy-compliance-status-for-each-cluster_{context}"]
+= Querying the policy compliance status for each cluster
+
+After you have created the validator inform policies for your clusters and pushed them to the
+the zero touch provisioning (ZTP) Git repository, you can check the status of each cluster for policy compliance.
+
+.Procedure
+
+. To query the status of the spoke clusters, use either the {rh-rhacm-first} web console or the CLI:
++
+* To query status from the {rh-rhacm} web console, perform the following actions:
++
+.. Click *Governance* -> *Find policies*.
+.. Search for *du-validator-policy*
+.. Click into the policy.
+
+* To query status using the CLI, run the following command:
++
+[source,terminal]
+----
+$ oc get policies du-validator-policy -n <namespace_for_common> -o jsonpath={'.status.status'} | jq
+----
++
+When all of the policies including the validator inform policy applied to the cluster become compliant,
+it means the ZTP installation and configuration for this cluster is complete.
+
+. To query the cluster violation/compliant status from the ACM web console, click
+*Governance* -> *Cluster violations*.
+
+. Check the validator policy compliant status for a cluster using the following commands:
++
+--
+.. Export the cluster name:
++
+[source,terminal]
+----
+$ export CLUSTER=<cluster_name>
+----
+
+.. Get the policy:
++
+[source,terminal]
+----
+$ oc get policies -n $CLUSTER | grep <validator_policy_name>
+----
+--
++
+Alternatively, you can use the following command:
++
+[source,terminal]
+----
+$ oc get policies -n <namespace-for-group> <validatorPolicyName> -o jsonpath="{.status.status[?(@.clustername=='$CLUSTER')]}" | jq
```

Comment:
Change the command to: 

oc get policies -n <namespace_for_group> <validator_policy_name> -o j-o jsonpath="{.status.status[?(@.clustername=='$CLUSTER')]}" | jq

Reply:
Also the string replacing mechanism is a bit inconsistent. We are using two different formats:

- <namespace-for-group> for user to replace it with the actual namespace
- $CLUSTER for user to replace it with the actual cluster name